engines/docker/resultset.go

```diff
@@ -103,7 +103,6 @@ func (r *resultSet) ExtractFolder(path string, handler engines.FileHandler) erro
 		if !strings.HasPrefix(name, "/") {
 			name = "/" + name // Ensure we always have an absolute path
 		}
-		name = name[len(path+"/"):]
```

Comment:
Is it possible that this should be `name = name[len(path):]` since we now have `path += "/"`